odoo/tools/query.py

```diff
@@ -69,6 +69,7 @@ def __init__(self, env, alias: str, table: (SQL | None) = None):
 
         # groupby, having, order, limit, offset
         self.groupby: SQL | None = None
+        self.extra_groupby: SQL | None = None
```

Comment:
After discussion, just name and type suggestion.

```suggestion
        self._order_groupby: list[SQL] | None = None
```